### DISCUSSION TITLE
Example code_relocation not compiling.

### DISCUSSION THREAD

**Discussion by AshwinKrishn** [zephyrproject-rtos/zephyr#38798] [Q&A] [answered]
**Describe the bug**
I am facing the same issue while building code_relocation example. 

Log : 
/home/ashwin/zephyrproject/zephyr/samples/application_development/code_relocation/prj.conf:1: warning: attempt to assign the value 'y' to the undefined symbol CODE_DATA_RELOCATION
Parsing /home/ashwin/zephyrproject/zephyr/Kconfig
Loaded configuration '/home/ashwin/zephyrproject/zephyr/boards/arm64/qemu_cortex_a53/qemu_cortex_a53_defconfig'
Merged configuration '/home/ashwin/zephyrproject/zephyr/samples/application_development/code_relocation/prj.conf'

error: Aborting due to Kconfig warnings

CMake Error at /home/ashwin/zephyrproject/zephyr/cmake/kconfig.cmake:269 (message):
  command failed with return code: 1
Call Stack (most recent call first):
  /home/ashwin/zephyrproject/zephyr/cmake/app/boilerplate.cmake:550 (include)
  /home/ashwin/zephyrproject/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:24 (include)
  /home/ashwin/zephyrproject/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:35 (include_boilerplate)
  CMakeLists.txt:5 (find_package)

-- Configuring incomplete, errors occurred!
FATAL ERROR: command exited with status 1: /usr/local/bin/cmake -DWEST_PYTHON=/usr/bin/python3 -B/home/ashwin/zephyrproject/zephyr/build -S/home/ashwin/zephyrproject/zephyr/samples/application_development/code_relocation -GNinja -DBOARD=qemu_cortex_a53 -DCMAKE_VERBOSE_MAKEFILE=ON

**To Reproduce**
1. west build -b qemu_cortex_a53 samples/application_development/code_relocation/  -- -DCMAKE_VERBOSE_MAKEFILE=ON
This can show the bug. 

**Expected behavior**
We can see that the .config is tried to be written but it does not based on already present .config. 
**Impact**
Unable to run the example

**Logs and console output**
If applicable, add console logs or other types of debug information
e.g Wireshark capture or Logic analyzer capture (upload in zip archive).
copy-and-paste text and put a code fence (\`\`\`) before and after, to help
explain the issue. (if unable to obtain text log, add a screenshot)

**Environment (please complete the following information):**
 - OS: Linux
 - Zephyr SDK - 
 - Commit SHA or Version used : 0d538447144cda0f15cde4c800efb353da541bc5

**Additional context**
NIL

**Comment by stephanosio** [comment, marked as answer]
https://github.com/zephyrproject-rtos/zephyr/blob/21d1ad3762302b3e461953df59430c77e0709274/arch/arm/Kconfig#L37-L39

`CODE_DATA_RELOCATION` is not supported on the `qemu_cortex_a53`.
